src/containers/CheckoutPage.jsx

```diff
@@ -483,6 +535,10 @@ class CheckoutPage extends React.Component {
               </div>
             </div>
             <div className="checkout-main-container">
+              <div className="profile-email-info">
+                <span className="feedback-label">{ email === '' && intl.get('email-validation') }</span>
+                <ProfileemailinfoMain profileInfo={profileData} onChange={this.fetchProfileData} />
```

Comment:
`profileData` could be undefined at this point and still throw an error which propagates to the component.